migtests/tests/oracle/co-db/fix-schema

```diff
@@ -5,16 +5,21 @@ set -x
 
 pushd ${EXPORT_DIR}/schema
 
-#Deleting views as the views are causing the import schema failures
-echo "Delete all views."
-echo "" > views/view.sql
+# Changing the datatype of product_details from bytea to TEXT as the check constraint doesn't work with bytea
 
-# removing typecase to json for bytea column(unsupported) so that CREATE TABLE succeeds
-# sed -i '/CONSTRAINT products_json_c CHECK/s/::pg_catalog\.json//g' sample2.sql
+sed -i -E 's/(product_details[[:space:]]+)bytea/\1TEXT/' tables/table.sql
```

Comment:
have you tested this replacement with both forms of SQL(DDL)
Formatted and unformatted? as that logic to classify as sharded/colocated(which causes the formatting/unformatting) might change.